README.md

```diff
@@ -1,222 +1,32 @@
 ![Terraform Provider Iterative](https://static.iterative.ai/img/cml/banner-terraform.png)
 
-# Terraform Iterative provider
-
 The Iterative Provider is a Terraform plugin that enables full lifecycle
```

Comment:
(Slightly off–topic)

[Not even they](https://www.terraform.io/docs/extend/index.html) agree on terminology, but mentioning both “provider” and “plugin” in the same line might be a bit too much. Given the repository name and the terminology on Terraform Registry, I would stick with “provider” everywhere.

Reply:
"The Iterative Provider is a Terraform provider" sounds too repetitive to me. They use "plugin" in their own docs for providers too - "The Terraform AWS provider is a plugin for Terraform that allows for the full lifecycle management of AWS resources. This provider is maintained internally by the HashiCorp AWS Provider team." [https://github.com/hashicorp/terraform-provider-aws](https://github.com/hashicorp/terraform-provider-aws)